test/TestImport.hs

```diff
@@ -24,4 +27,28 @@ withApp = before $ do
         ["config/test-settings.yml", "config/settings.yml"]
         []
         ignoreEnv
-    makeFoundation settings
+    foundation <- makeFoundation settings
+    wipeDB foundation
+    return foundation
+
+-- This function will truncate all of the tables in your database.
+-- 'withApp' calls it before each test, creating a clean environment for each
+-- spec to run in.
+wipeDB :: App -> IO ()
+wipeDB app = do
+    runDBWithApp app $ do
+        tables <- getTables
+        let queries = map ("TRUNCATE TABLE " ++ ) tables
+
+        -- In MySQL, a table cannot be truncated if another table references it via foreign key.
+        -- Since we're wiping both the parent and child tables, though, it's safe
+        -- to temporarily disable this check.
+        rawExecute "SET foreign_key_checks = 0;" []
+        forM_ queries (\q -> rawExecute q [])
+        rawExecute "SET foreign_key_checks = 1;" []
+    return ()
+
+getTables :: MonadIO m => ReaderT SqlBackend m [Text]
+getTables = do
+    tables <- rawSql "SHOW TABLES;" []
```

Comment:
I'm OK leaving it as-is for now. I just realized one last issue: table names need to be escaped. I believe you can use [connEscapeName](http://www.stackage.org/haddock/lts-1.0/persistent-2.1.1.3/Database-Persist-Sql.html#v:connEscapeName) for that. Once that's done, I think this can be merged in. Thanks!